.github/ISSUE_TEMPLATE/bump_gmt_checklist.md

```diff
@@ -17,11 +17,12 @@ assignees: ''
 **To-Do for bumping the GMT version in CI**:
 
 - [ ] Bump the GMT version in CI (1 PR)
+  - [ ] Update `environment.yml`
+  - [ ] Update `ci/requirements/docs.yml`
```

Comment:
> So no need to include the other `ci/requirements/*.yml` files because the GMT version isn't pinned?

Yes, GMT is not pinned in other requirement files. The only exception is `ci/requirements/docs.yml` because it's also used by ReadTheDocs.